src/Formatter/DateTimeFormatter.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Formatter;
+
+use Symfony\Component\PropertyAccess\PropertyPath;
+
+final class DateTimeFormatter implements DataFormatterInterface
+{
+    /** @var null|string */
+    private $dateTimeFormat = null;
+
+    /**
+     * DateTimeFormatter constructor.
+     * @param null|string $dateTimeFormat
+     */
+    public function __construct($dateTimeFormat = 'r')
+    {
+        $this->dateTimeFormat = $dateTimeFormat;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function supports($data)
+    {
+        return ($data instanceof \DateTime);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function format($data, PropertyPath $propertyPath)
+    {
+        return $data->format($this->dateTimeFormat);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getPriority()
+    {
+        return PHP_INT_MAX-2;
```

Comment:
Please make room for other formatters, by using 10, 20, 30, instead of 1, 2, 3, and maybe PHP_INT_MAX is overkill :laughing: